### DISCUSSION TITLE
Can we give custom styling to the sectors in style manager?

### DISCUSSION THREAD

**Discussion by detronetdip** [GrapesJS/grapesjs#5260] [Q&A] [answered]
Hi, I want to build a customer editor for my product on top of GrapesJs I have al ready worked a bit on this now I want to give custom styling to each sectors in style manager and want to change the way they look. 

By default the all appear in a collapsible manner but I want to change it, like probably instead of collapsible I would like to choose tab style or may be  move different sectors to different positions like that.

Is it possible?

**Comment by artf** [comment, marked as answer]
Hi @detronetdip did you check already the [Style Manager doc page](https://grapesjs.com/docs/modules/Style-manager.html)?
You can also replace entirely the default UI with [your own Style Manager](https://grapesjs.com/docs/modules/Style-manager.html#custom-style-manager)

**Reply by detronetdip** [reply]
I am still facing an issue to create my very own style manager UI for the editor. Can you please help me out?

**Comment by detronetdip** [comment]
@artf thanks for the heads up it's really helpful
